.eslintrc.js

```diff
@@ -15,7 +15,13 @@ module.exports = {
     jest: true
   },
   globals: {
-    ENV: true
+    ENV: true,
+    after: true,
+    before: true,
+    browser: true,
```

Comment:
What are your thoughts on `browser` given this config is used on node projects too. Should we add a separate node profile that sets this to `false`?

Reply:
Usually it's one config for the whole project though, not sure if we want to start splitting it and creating extra setup for that 🤔